### PR TITLE
Increase memory devoted to tests

### PR DESCRIPTION
**Pull request by chibenwa**
We experience many OutOfMemory exception on our CI and 1GiB is a rather
low limit. Increasing it might solve the issue.

### CODE REVIEW

File: server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/DownloadStepdefs.java [hunk 1]
```diff
@@ -110,7 +110,7 @@ public void appendBigMessageToMailbox(String user, String mailbox, String messag
 
         ComposedMessageId composedMessageId = mainStepdefs.mailboxProbe.appendMessage(user, mailboxPath,
             AppendCommand.from(new ByteArrayInputStream(
-                Strings.repeat("header: 0123456789\r\n", 1024 * 1024)
+                Strings.repeat("header: 0123456789\r\n", 128 * 1024)
```
Comment (trantienduchn) [inline server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/DownloadStepdefs.java]: I guess `DownloadGet.feature` uses this test. The original scenario:
```
  Scenario: Getting a message blob previously stored
    Given "alice@domain.tld" mailbox "INBOX" contains a big message "1"
    When "alice@domain.tld" downloads "1"
    Then the blob size is 20971520
```
Should we update blob size to fit with the updated code.

Comment (trantienduchn) [reply server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/DownloadStepdefs.java]: The purpose of that tests is testing with a big message, does reducing message size satisfy the original purpose?

Comment (chibenwa) [reply server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/DownloadStepdefs.java]: No, as the name does not suggest it, it do ensure that COntent-Length is specified before download.

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
```

[219f7452299030f936e13e042efaec83e591abcf]   AbstractNotificationRegistryTest.registerShouldWorkWithExpiracyDate:72 expected:<[tru]e> but was:<[fals]e>
```

test this please

**Comment by chibenwa** [comment]
```
[219f7452299030f936e13e042efaec83e591abcf] Error executing docker rm --force=true --volumes=true git-publish-surefire-1757d6e3-7607-4528-b7f9-1e46ad8a8177 : no known node for /home/jenkins/build/workspace/710
```

yoda failure

test this please

**Comment by chibenwa** [comment]
IMO could be considered as :green_apple:  : test suite passed, yoda failed to play deployment tests, but this PR should not have impact with deployment...

**Comment by chibenwa** [comment]
https://ci.linagora.com/linagora/lgs/openpaas/james/issues/675 proposes to have a test for large mail sending.

File: pom.xml [hunk 2]
```diff
@@ -2781,7 +2781,7 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>2.19.1</version>
                     <configuration>
-                        <argLine>-Xms512m -Xmx1024m</argLine>
+                        <argLine>-Xms512m -Xmx4096m</argLine>
```
[the remark this answers is not in the document]
Comment (chibenwa) [reply pom.xml]: Reverted. Fixing the Content-Length test seems should be enough.

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**